src/numberGame.js

```diff
@@ -0,0 +1,56 @@
+//Import functions to reference
+import compareNumbers from './guessnumber.js';
+
+//Reference needed DOM elements
+const checkAnswer = document.getElementById('check-answer');
+const guess = document.getElementById('user-guess');
+const resultsSection = document.getElementById('results-section');
+const guessResponse = document.getElementById('guess-response');
+const guessesLeft = document.getElementById('guesses-left');
+
+
+//Initialization
+// Math function from MDN
+const correctNumber = Math.floor(Math.random() * 20);
+
+let guessCount = 4;
+
+//Event Handlers
+checkAnswer.addEventListener('click', () => {
+    guessCount -= 1;
+
+    // If the user guesses correct answer
+    if(compareNumbers(Number(guess.value), correctNumber) === 0) {
+        resultsSection.classList.remove('hidden');
+        guessResponse.textContent = '🎉 🏄🏻‍♀️ You got it! Way to go! 🎉 🏄🏻‍♀️';
+        guessesLeft.classList.add('hidden');
+        checkAnswer.disabled = true;
+    }
+    
```

Comment:
generally your will want to remove this line of white space between the `else if` statements, and when you make your final PR, go through and remove all comments and console.logs. (you will always have access to them in your prior commit history), these are just little nit-picky things, your code looks great!